frontend/src/models/dashboardsModel.tsx

```diff
@@ -175,6 +207,10 @@ export const dashboardsModel = kea<dashboardsModelType>({
             toast(`Dashboard "${dashboard.name}" created!`)
         },
 
+        copyDashboardSuccess: ({ dashboard }) => {
+            toast(`Dashboard copied to "${dashboard.name}"!`)
```

Comment:
Nice bit of UX toast here! The wording is a little confusing to me because `copying to` feels like the dashboard is being placed inside of another location. I think something like `Dashboard copied as ${dashboard.name}` would sound better